src/components/CVModal/CVModalButtonBar/index.js

```diff
@@ -3,15 +3,16 @@ import { PropTypes as PT } from 'prop-types';
 import { CancelModalButton, DownloadModalButton } from 'Components';
 import StyledButtonBar from './styled';
 
-export const CVModalButtonBar = ({ handleCancelClick }) => (
+export const CVModalButtonBar = ({ handleCancelClick, cvData }) => (
```

Comment:
I'm seeing `cvData` used a lot. It's used for a href and passed through a few areas. What does it contain?

Reply:
It contains the endpoint for downloading the employee CV. I've changed it to cvURL which hopefully is more descriptive.

Reply:
Perfect.